subcommands/devices/list.go

```diff
@@ -228,7 +212,7 @@ func doList(cmd *cobra.Command, args []string) {
 		"uuid":        deviceUuid,
 	}
 	if len(args) == 1 {
-		filterBy["name_ilike"] = sqlLikeIfy(args[0])
```

Comment:
Won't it break customer clients using the API directly as well as the old `fioctl` versions. I mean the `name_ilike` removal?

Reply:
The `name_ilike` was not removed on the API side...
It is still supported there, but was deprecated quite a few months ago.
So, no clients were broken.

It is only the Fioctl finally moving to a newer API argument.

Reply:
I've already checked the backend part and see that both params are supported.